### PR TITLE
Hindi translation of pipeline_tutorial.md

### PR DESCRIPTION
**Pull request by AaryaBalwadkar**
# What does this PR do?
This PR translates the pipeline_tutorial.md in Hindi language and mentions it in _toctree.yml in doc/source/hi/ folder. Thus  Fixes # 26787

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).

## Who can review?

@stevhliu and @MKhalusova please review this PR.

### CODE REVIEW

**Comment by MKhalusova** [comment]
Hi @AaryaBalwadkar! Thank you for working on translating the doc page into Hindi. 
From what I see, this is the first time a doc page has been translated into Hindi. As such, you'll need to modify two more files in order to build the docs in a new language - `.github/workflows/build_documentation.yml` and `.github/workflows/build_pr_documentation.yml`.  

Add the two-letter code for your language to the list of languages. You can look up the code [here](https://www.loc.gov/standards/iso639-2/php/code_list.php).

**Comment by AaryaBalwadkar** [comment]
Done the modification in both the files. Please have a look.

File: docs/source/hi/pipeline_tutorial.md
Comment (hakunamatata1997) [inline docs/source/hi/pipeline_tutorial.md:1]: minor changes needed

**Review by hakunamatata1997 — COMMENTED** [review]
you mentioned device=n instead of device=0

File: docs/source/hi/pipeline_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+transcriber = pipeline(model="openai/whisper-large-v2", my_parameter=1)
+
+out = transcriber(...)  # This will use `my_parameter=1`.
+out = transcriber(..., my_parameter=2)  # This will override and use `my_parameter=2`.
+out = transcriber(...)  # This will go back to using `my_parameter=1`.
+```
+
+आइए 3 महत्वपूर्ण बातों पर गौर करें:
+
+## उपकरण
```
Comment (stevhliu) [inline docs/source/hi/pipeline_tutorial.md:107]: This should be a level 3 header

```suggestion
### उपकरण
```

File: docs/source/hi/pipeline_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+निम्नलिखित कोड स्वचालित रूप से सभी डिवाइसों में मॉडल भार को लोड और संग्रहीत करता है:
+
+```py
+transcriber = pipeline(model="openai/whisper-large-v2", device_map="auto")
+```
+
+ध्यान दें कि यदि `device_map='auto'` पारित हो गया है, तो अपनी `pipeline` को चालू करते समय `device=device` तर्क जोड़ने की कोई आवश्यकता नहीं है क्योंकि आपको कुछ अप्रत्याशित व्यवहार का सामना करना पड़ सकता है!
+
+## बैच का आकार
```
Comment (stevhliu) [inline docs/source/hi/pipeline_tutorial.md:132]:
```suggestion
### बैच का आकार
```

File: docs/source/hi/pipeline_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+...     images="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/pipeline-cat-chonk.jpeg"
+... )
+>>> preds = [{"score": round(pred["score"], 4), "label": pred["label"]} for pred in preds]
+>>> preds
+[{'score': 0.4335, 'label': 'lynx, catamount'}, {'score': 0.0348, 'label': 'cougar, puma, catamount, mountain lion, painter, panther, Felis concolor'}, {'score': 0.0324, 'label': 'snow leopard, ounce, Panthera uncia'}, {'score': 0.0239, 'label': 'Egyptian cat'}, {'score': 0.0229, 'label': 'tiger cat'}]
+```
+
+## पाठ पाइपलाइन
+
+NPL कार्यों के लिए [`pipeline`] का उपयोग करना व्यावहारिक रूप से समान है।
```
Comment (stevhliu) [inline docs/source/hi/pipeline_tutorial.md:248]:
```suggestion
NLP कार्यों के लिए [`pipeline`] का उपयोग करना व्यावहारिक रूप से समान है।
```

File: docs/source/hi/pipeline_tutorial.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+...     candidate_labels=["urgent", "not urgent", "phone", "tablet", "computer"],
+... )
+{'sequence': 'I have a problem with my iphone that needs to be resolved asap!!', 'labels': ['urgent', 'phone', 'computer', 'not urgent', 'tablet'], 'scores': [0.504, 0.479, 0.013, 0.003, 0.002]}
+```
+
+## बहुविध पाइपलाइन
+
+[`pipeline`] एक से अधिक तौर-तरीकों का समर्थन करती है। उदाहरण के लिए, एक दृश्य प्रश्न उत्तर (VQA) कार्य पाठ और छवि को जोड़ता है। अपनी पसंद के किसी भी छवि लिंक और छवि के बारे में कोई प्रश्न पूछने के लिए स्वतंत्र महसूस करें। छवि एक URL या छवि का स्थानीय पथ हो सकती है।
+
+उदाहरण के लिए, यदि आप इस [Invoice image](https://huggingface.co/spaces/impira/docquery/resolve/2359223c1837a7587402bda0f2643382a6eefeab/invoice.png) का उपयोग करते हैं:
```
Comment (stevhliu) [inline docs/source/hi/pipeline_tutorial.md:267]:
```suggestion
उदाहरण के लिए, यदि आप इस [invoice image](https://huggingface.co/spaces/impira/docquery/resolve/2359223c1837a7587402bda0f2643382a6eefeab/invoice.png) का उपयोग करते हैं:
```

**Review by stevhliu — COMMENTED** [review]
Thanks for the translation!

Can you include the changes you made in #26855 in this PR? The doc-builder test on that PR is failing because it can't find the `toctree` to build the docs.

**Comment by AaryaBalwadkar** [comment]
Ok

**Comment by AaryaBalwadkar** [comment]
Made the changes. Please have a look

**Comment by stevhliu** [comment]
Thanks! Can you include the changes you made in https://github.com/huggingface/transformers/pull/26855 in this PR as well?

**Review by stevhliu — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**